internal/ci/ethereum_test

```diff
@@ -2,6 +2,9 @@
 
 set -e
 
+# Kill all background processes when this shell script finishes
+trap "kill -- -$$" SIGINT SIGTERM EXIT
```

Comment:
👍

Reply:
Well, now that we know this is supposed to be `trap "kill -- -$$ || true" SIGINT SIGTERM EXIT`, let's 👍.